app/concierge/concierge/urls.py

```diff
@@ -16,17 +16,33 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
-from django.urls import path
+from django.contrib.auth.decorators import permission_required
+from django.urls import path, include
+from django.views.decorators.cache import cache_page
+from django.views.generic import TemplateView
+from mycore.views import api_serializer, RoomView, TenantView, JournalView, RoomListView, RoomDetailView, \
+    TenantListView, TenantDetailView, JournalListView, JournalDetailView
```

Comment:
if an import is too long, cover it with tuple like this: 
`from something import (one, two, three)`